setup.cfg

```diff
@@ -47,6 +47,8 @@ dev =
     coverage
     elasticsearch-dsl~=7.4
     pytest
+export =
```

Comment:
Is the scenario for this `extra` that sometimes we want to export the ES configuration for ES build as a standalone process, but most of the time the gdc-models are used like a library like in the gdcapi?

Reply:
Yup. We only want to install/run export when we are working within the repo currently. When we install the library elsewhere we are just trying to access the models as they are